module-2/backend/week_one.md

```diff
@@ -5,30 +5,100 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+ Get: retreive/display info, Post: display new info, Put:save new info, Delete: delete info, Patch: update/fix?
+
 2. What is Sinatra?
+
+ Sinatra is a web app framework.
+
 4. What is MVC?
+
+ MVC is models, views, controllers. It is a pattern put in place to organize information based on functionality.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+ Because it's restful.
+ 
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+ params

```

Comment:
The params hash isn't available to us in our view. We were looking for instance variables.